### DISCUSSION TITLE
Is juicefs server compatible with nfs?

### DISCUSSION THREAD

**Discussion by ggaaooppeenngg** [juicedata/juicefs#1711] [Q&A] [answered]
**What would you like to be added**:

**Why is this needed**:

**Comment by davies** [comment, marked as answer]
nfs is just a protocol, both of JuiceFS and NFS (NAS using NFS) are compatible to POSIX API, so you can use JuiceFS to replace NFS.

Btw, you can use nfsd over JuiceFS to provide NFS protocol for JuiceFS.
